src/sidebar/services/persisted-defaults.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * A service for reading and persisting convenient client-side defaults for
+ * the (browser) user.
+ */
+
+const DEFAULT_KEYS = {
```

Comment:
Using a single service to manage both ends of this obviates the need for a slightly-silly util (from previous attempt)